Kernel/API/Syscall.h

```diff
@@ -364,10 +364,10 @@ struct SC_create_thread_params {
     unsigned int stack_size = 1 * MiB;         // Equal to Thread::default_userspace_stack_size
     void* stack_location;                      // nullptr means any, o.w. process virtual address
 #    if ARCH(X86_64)
+    FlatPtr rdx;
```

Comment:
This shouldn't be in a PR titled "Kernel/aarch64". Put it in a separate PR, maybe?

Reply:
Oh! Missed that completely, thanks! Iʼve messed commits across my pull requests! This changes landed in #17205, where they belong.